### PR TITLE
[#329] Add storage building instructions for LambdaDAO

### PR DESCRIPTION
**Pull request by sras**
## Description

Problem: We missed to add storage building instructions for LambdaDAO
and variants to 'building.md'.

Solution: Add the instructions to 'building.md'.

## Related issue(s)

Resolves #

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

[//]: # (Add more docs here if you have them in the repository)
- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)
    - Haddock

#### Stylistic guide (mandatory)

- [x] My commits comply with [the following policy](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: docs/building.md [hunk 1]
```diff
@@ -233,6 +233,105 @@ The `admin_address`, `guardian_address`, `governance_token_address`,
 `start_level` and `governance_token_id` are required values. The rest of the
```
Comment (pasqu4le) [inline docs/building.md:233]: I just noticed that there is a mistake in the `make` call example for `treasuryDAO`: the `freeze_history` option is specified twice, please fix that as well.

File: docs/building.md [hunk 2]
```diff
@@ -233,6 +233,105 @@ The `admin_address`, `guardian_address`, `governance_token_address`,
...
+  guardian_address=KT1QbdJ7M7uAQZwLpvzerUyk7LYkJWDL7eDh \
+  governance_token_address=KT1RdwP8XJPjFyGoUsXFQnQo1yNm6gUqVdp5 \
+  governance_token_id=0n \
+  frozen_scale_value=1n \
+  frozen_extra_value=0n \
+  max_proposal_size=100n \
+  slash_scale_value=1n \
+  slash_division_value=1n \
+  min_xtz_amount=0mutez \
+  max_xtz_amount=100mutez \
```
Comment (pasqu4le) [inline docs/building.md:253]:
```suggestion
```
these seem unnecessary

File: docs/building.md [hunk 3]
```diff
@@ -233,6 +233,105 @@ The `admin_address`, `guardian_address`, `governance_token_address`,
...
```
Comment (pasqu4le) [inline docs/building.md:286]: I believe that the should be used, but looking at the `Makefile` they don't seem to be? Is this a mistake?

File: docs/building.md [hunk 4]
```diff
@@ -233,6 +233,105 @@ The `admin_address`, `guardian_address`, `governance_token_address`,
...
```
Comment (pasqu4le) [inline docs/building.md:319]: Same question as for `LambdaRegistryDAO`

**Review by pasqu4le — CHANGES_REQUESTED** [review]

File: docs/building.md (hunk 3 above)
Comment (sras) [reply docs/building.md:286]: I think it is a mistake. And it seems that tests need to be expanded to include this check for these variants.

Comment (sras) [reply docs/building.md:278]: The configuration values passed from makefile did not appeared to be used even for original registry/treasury. Have fixed it now.

File: docs/building.md (hunk 1 above)
Comment (sras) [reply docs/building.md:232]: Fixed.

**Review by pasqu4le — APPROVED** [review]
One more nitpick, but the rest LGTM

File: Makefile [hunk 5]
```diff
@@ -282,33 +289,48 @@ $(OUT)/lambdaregistryDAO_storage.tz: src/**
...
-          ; proposal_expired_level = { blocks = ($(proposal_expired_level) : nat) }\
-          ; fixed_proposal_fee_in_token = ($(fixed_proposal_fee_in_token) : nat) \
-          ; quorum_threshold = { numerator = (($(quorum_threshold) : nat) * quorum_denominator)/100n } \
-          ; quorum_change = { numerator = (($(quorum_change) : nat) * quorum_denominator)/100n } \
-          ; max_quorum_change = { numerator = (($(max_quorum_change) : nat) * quorum_denominator)/100n } \
-          ; governance_total_supply = ($(governance_total_supply) : nat) \
-          } \
+      src/base_DAO.mligo -e base_DAO_contract "default_lambda_registry_DAO_storage( \
+        { base_data = \
+					{ admin = (\"$(call require_defined,admin_address)\" : address) \
```
[the remark this answers is not in the document]
Comment (sras) [reply Makefile]: Have used spaces instead of tabs for indenting new lines.

**Pull request merged**